block_my_feedback.php

```diff
@@ -38,7 +38,7 @@ public function init() {
 
         if (!isset($USER->firstname)) {
             $this->title = get_string('pluginname', 'block_my_feedback');
-        } else if (self::is_teacher()) {
+        } else if ($this->is_teacher()) {
```

Comment:
`is_teacher()` is static so I'm not sure why this has been changed, I would expect this to be `self::is_teacher()` as before.

This post [says](https://dev.to/ryancco/the-power-of-static-in-php-4130#method-signature): _While static methods can be referred to using the object operator (->) it is recommended to use the scope resolution operator (::) as the alternative is deprecated and may be removed in the future._